src/client.c

```diff
@@ -68,11 +102,34 @@ int send_request(int fd, char *hostname, char *port, char *path)
   char request[max_request_size];
   int rv;
 
-  ///////////////////
-  // IMPLEMENT ME! //
-  ///////////////////
+  // time_t time1 = time(NULL);
+  // struct tm *localtime1 = localtime(&time1);
 
-  return 0;
+
+  
+  // stringify the passed data into the request packet buffer
+  int request_length = sprintf(request,
+      "GET PATH: %s\n"
```

Comment:
According to the `README.md`, our HTTP request should look like this:

```
GET /path HTTP/1.1
Host: hostname:port
Connection: close
```

So we can do something like this:

```c
"GET /%s HTTP/1.1\n"
"Host: %s:%s\n"
"Connection: close\n"
```